### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by ozymand1a** [mlflow/mlflow#5404]
Thank you for this repo. It's great and now I am using it for my tasks. Recently I encountered the following problem: 

I try to push my figure to the server with this method. 
```
from mlflow.tracking.client import MlflowClient
    for logger in loggers:
        if logger.__class__ == MlflowClient:
            logger: MlflowClient = logger
            logger.log_figure(run_id, figure, artifact_file)
```
But then I have error:

`AttributeError: module 'matplotlib' has no attribute 'figure'`

Then I find that moment at code here https://github.com/mlflow/mlflow/blob/master/mlflow/tracking/client.py:

<img width="602" alt="image" src="https://user-images.githubusercontent.com/54213286/155188250-afc1ddfc-c834-465c-a452-30b9b4691fae.png">

I reproduce this problem by next code:

![telegram-cloud-photo-size-2-5233537355665750798-y](https://user-images.githubusercontent.com/54213286/155187632-92e6caa4-588d-4a4f-a2da-8519bbc14258.jpg)

When I change this moment at the package, my logging starts working perfectly.

I am sure that I have this bug by me, so how can I fix it without changing the MLFlow source code?

**Comment by dbczumar** [comment]
@ozymand1a What version of matplotlib are you using?

**Comment by WeichenXu123** [comment]
I think the issue is if only import matplotlib, it does not import submodule "matplotlib.figure", we should explicitly import "matplotlib.figure" submodule

**Comment by WeichenXu123** [comment]
create a fix https://github.com/mlflow/mlflow/pull/5406

**Comment by ozymand1a** [comment]
> @ozymand1a What version of matplotlib are you using? We recommend using the latest one (`3.5.1`).

yes, I use this version, but it doesn't help me

**Comment by dbczumar** [comment]
@ozymand1a Got it! Thank you! The fix has been merged (#5406) and will be included in the next release.

**Comment by ozymand1a** [comment]
> @ozymand1a Got it! Thank you! The fix has been merged (#5406) and will be included in the next release.

@dbczumar Thank you!
